### DISCUSSION TITLE
2024/10/28/ICPC%E6%88%90%E9%83%BD%E7%AB%99%E6%B8%B8%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [ShwStone/shwblog#78] [Announcements]
# 2024/10/28/ICPC%E6%88%90%E9%83%BD%E7%AB%99%E6%B8%B8%E8%AE%B0/

https://shwstone.cn/2024/10/28/ICPC%E6%88%90%E9%83%BD%E7%AB%99%E6%B8%B8%E8%AE%B0/

**Comment by Panda-Ghost** [comment]
shw 还是太强了

**Comment by THEzsc** [comment]
shw还是太强了
